test/services/WhatYouOweServiceSpec.scala

```diff
@@ -49,7 +49,12 @@ class WhatYouOweServiceSpec extends TestSupport with FeatureSwitching {
   val mockFinancialDetailsService: FinancialDetailsService = mock[FinancialDetailsService]
   val mockIncomeTaxViewChangeConnector: IncomeTaxViewChangeConnector = mock[IncomeTaxViewChangeConnector]
 
-  object TestWhatYouOweService extends WhatYouOweService(mockFinancialDetailsService, mockIncomeTaxViewChangeConnector)
+  object mockDateService extends DateService() {
+    override def getCurrentDate: LocalDate = LocalDate.parse(s"${LocalDate.now.getYear.toString}-04-01")
+    override def getCurrentTaxYearEnd(currentDate: LocalDate): Int = 2022
```

Comment:
Sorry I meant the year on this line as well. Rather than hard coding its better to use LocalDate.noew(),getYear

Reply:
Done, thanks